packages/pluginutils/src/composable-filters.ts

```diff
@@ -140,6 +148,10 @@ export function code(pattern: StringOrRegExp): Code {
   return new Code(pattern);
 }
 
+export function query(key: string, pattern: StringOrRegExp | boolean): Query {
+  return new Query(key, pattern);
```

Comment:
Maybe this should be part of Id? (this only matches the id right?) In the future, we might add a way to match `importer` and probably good to take that into consideration.